cmd/codecrafters/main.go

```diff
@@ -26,10 +26,12 @@ EXAMPLES
   $ codecrafters test              # Run tests without committing changes
   $ codecrafters test --previous   # Run tests for the current stage and all previous stages without committing changes
   $ codecrafters submit            # Commit changes & submit to move to next step
+  $ codecrafters docs              # View current stage instructions
```

Comment:
Let's remove this from the top-level examples too, isn't going to be a commonly used command